src/neovim.rs

```diff
@@ -96,17 +103,31 @@ where
     (req, fut)
   }
 
-  async fn send_msg(
+  /// Will panic if args is serialized into something that is not an array
```

Comment:
Where does that happen? How can the caller ensure this doesn't happen? Could we statically guarantee that somehow?